tests/basic/storage_pool_test.go

```diff
@@ -5188,3 +5190,195 @@ var _ = Describe("{ResizePoolDrivesInDifferentSize}", func() {
+			}
+			log.InfoD("Current Size of the pool %s is %d", poolIDToResize, poolToBeResized.TotalSize/units.GiB)
+			// Delete the Volume that was ~190G before the pool expand begins
+			// Iterate through the contexts, get the volumes and then get the matching ID
+		gotContext:
+			for _, l := range contexts {
+				vols, err := Inst().S.GetVolumes(l)
+				dash.VerifyFatal(err, nil, "Verify if able to get the app for the volume that is filled approx 200G")
+				for _, vol := range vols {
+					if vol.ID == volID {
+						contextToDel = l
+						break gotContext
+					}
+				}
+			}
+			err = Inst().V.ExpandPool(poolIDToResize, api.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, expectedSize)
+			dash.VerifyFatal(err, nil, "Pool expansion init successful?")
+			// Destroy the context
+			err = Inst().S.Destroy(contextToDel, nil)
+			dash.VerifyFatal(err, nil, "Verify the successful delete context of the volume which had ~190 G usage")
+			log.InfoD("Going to delete the volume, by deletion of Namespace")
+			TearDownContext(contextToDel, map[string]bool{
+				SkipClusterScopedObjects:                    false,
+				scheduler.OptionsWaitForResourceLeakCleanup: true,
+				scheduler.OptionsWaitForDestroy:             true,
+			})
+			dash.VerifyFatal(err, nil, "Verify the successful delete of the volume which had ~190 G usage")
+			resizeErr := waitForPoolToBeResized(expectedSize, poolIDToResize, isjournal)
+			dash.VerifyFatal(resizeErr, nil, fmt.Sprintf("Expected new size to be '%d' or '%d'", expectedSize, expectedSizeWithJournal))
+		})
+		// Make sure to remove the deleted context and validate the other apps
+		for _, l := range contexts {
+			if l.App.Key != contextToDel.App.Key {
+				newContexts = append(newContexts, l)
+			}
+		}
+		stepLog = "Ensure that new pool has been expanded to the expected size and also check the pool expand alert"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			ValidateApplications(newContexts)
+			resizedPool, err := GetStoragePoolByUUID(poolIDToResize)
+			log.FailOnError(err, fmt.Sprintf(" Failed to get pool using UUID %s", poolIDToResize))
+			newPoolSize := resizedPool.TotalSize / units.GiB
+			isExpansionSuccess := false
+			if newPoolSize == expectedSize || newPoolSize == expectedSizeWithJournal {
+				isExpansionSuccess = true
+			}
+			dash.VerifyFatal(isExpansionSuccess, true, fmt.Sprintf("Expected new pool size to be %v or %v, got %v", expectedSize, expectedSizeWithJournal, newPoolSize))
+			log.Infof("Check the alert for pool expand for pool uuid %s", poolIDToResize)
+			// Get the node to check the pool show output
+			n := node.GetStorageDriverNodes()[0]
+			// Below command to change when PWX-28484 is fixed
+			cmd := "pxctl alerts show| grep -e POOL"
+
+			// Execute the command and check the alerts of type POOL
+			out, err := Inst().N.RunCommandWithNoRetry(n, cmd, node.ConnectionOpts{
+				Timeout:         2 * time.Minute,
+				TimeBeforeRetry: 10 * time.Second,
+			})
+			if err != nil {
```

Comment:
if check is not required as log.onFailError will handle it

Reply:
done